application/views/surveyAdministration/activateSurvey_view.php

```diff
@@ -255,10 +254,21 @@ function alertPrivacy()
                     </div>
                 </div>
             </div>
-
+            
+            <!-- Tip -->
             <div class='row'>
                 <div class='col-sm-8 col-sm-offset-2'>
-                    <div class="alert alert-info"><?php eT("Tip: Please note that you need to deactivate a survey if you want to add or delete groups/questions or change any of the settings above. The changes will cause all collected data from respondents to be moved and archived."); ?></div>
+                    <div class="alert alert-info">
```

Comment:
Yeah, this won't work. :) Not all languages have the same order of verbs, nouns, etc etc.